src/DbUtils.php

```diff
@@ -493,8 +493,8 @@ public function getItemForItemtype($itemtype)
     /**
      * Count the number of elements in a table.
      *
-     * @param string|array $table     table name(s)
-     * @param array        $condition array of criteria
+     * @param string|array   $table     table name(s)
+     * @param ?string|?array $condition array of criteria
```

Comment:
I am not even sure it would work. We never cast condition to an array, and then we do `$condition['COUNT'] = 'cpt';`. IMHO, if `$condition` is a string, it would probably fail. We sould check this, maybe the `if (!is_array($condition)) {}` block is actually some dead code.

Same remark applies for other changes in this file.

Reply:
"IMHO, if $condition is a string, it would probably fail" well... Iterator still accepts string as a condition, so it should still work.

I'm not in favor to change logic for a simple phpdoc/CS issue.